src/main/java/org/example/service/JsonService.java

```diff
@@ -0,0 +1,27 @@
+package org.example.service;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import org.example.model.Developer;
+
+import java.util.List;
+
+
+public class JsonService {
+    private static JsonService INSTANCE;
```

Comment:
Check the naming conventions.
When the variable title should be written in caps.

Reply:
fixed